### PR TITLE
Added troubleshooting information and issue templates for bug report and feature request

### PR DESCRIPTION
**Pull request by lintongj**
The following two changes are included.
* Added troubleshooting information 
* Added issue templates for bug report and feature request

### CODE REVIEW

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/456/

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/457/

**Review by deepakkinni — APPROVED** [review]
lgtm

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Troubleshooting
+
+To troubleshoot issues in velero-plugin-for-vsphere, the following information is expected.
+
+## Environment
+
+The following environment information will also be **required**.
+
+- velero-plugin-for-vsphere version
+- Kubernetes cluster flavor
```
Comment (xing-yang) [inline docs/troubleshooting.md:10]: List supported flavors.

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/459/

**Review by xing-yang — APPROVED** [review]
LGTM

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/460/

**Pull request merged**